src/resources/games-toolkit.md

```diff
@@ -1,241 +1,382 @@
+games.
 
-[Flutter + Flame codelab]: {{site.codelabs}}/codelabs/flutter-flame-game
+* VGV created the [I/O Pinball game][pinball-game] using the Flame engine.
+  To learn about this game,
+  check out [I/O Pinball Powered by Flutter and Firebase][] on Medium
+  and [play the game][pinball-game] in your browser.
 
-In fact, the people at Very Good Ventures wrote a
-[Pinball game][pinball-game] for Google I/O 2022
-using the Flame engine. For more information, check out
-[I/O Pinball Powered by Flutter and Firebase][],
-a free article on Medium and [play the game][pinball-game].
+* The Flutter team created [I/O Flip][flip-game], a virtual [CCG].
+  To learn more about I/O Flip,
+  check out [How It’s Made: I/O FLIP adds a twist to a classic card game with generative AI][flip-blog]
+  on the Google Developers blog and [play the game][flip-game] in your browser.
 
 {% comment %}
 Some examples of successful mobile videogames
-built with Flutter and Flame include 
+built with Flutter and Flame include
 [Tomb Toad][] and [Air Hockey][].
 {% endcomment %}
 
-[Air Hockey]: https://play.google.com/store/apps/details?id=com.ignacemaes.airhockey
-[flame-demo]: https://dartpad.dev/?id=3e52ca7b51ba15f989ad880b8b3314a2
-[flame-discord]: https://discord.gg/5unKpdQD78
-[flame-package]: {{site.pub}}/packages/flame
-[I/O Pinball Powered by Flutter and Firebase]: {{site.medium}}/flutter/di-o-pinball-powered-by-flutter-and-firebase-d22423f3f5d
-[pinball-game]: https://pinball.flutter.dev/#/
-[Tomb Toad]: https://play.google.com/store/apps/details?id=com.crescentmoongames.tombtoad
-
 ## Other resources
 
-Once you feel ready to go beyond the games template,
-you might want to investigate other resources
-recommended by the community.
-Bolded items in the following table are used in the games template:
+Once you feel ready to go beyond these games templates,
+investigate other resources that our community recommended.
+
+{% assign pkg-icon = '<span class="material-symbols-outlined">package_2</span>' %}
+{% assign doc-icon = '<span class="material-symbols-outlined">quick_reference_all</span>' %}
+{% assign codelab = '<span class="material-symbols-outlined">science</span>' %}
+{% assign engine = '<span class="material-symbols-outlined">manufacturing</span>' %}
+{% assign tool-icon = '<span class="material-symbols-outlined">handyman</span>' %}
+{% assign recipe-icon = '<span class="material-symbols-outlined">book_5</span>' %}
+{% assign assets-icon = '<span class="material-symbols-outlined">photo_album</span>' %}
+{% assign api-icon = '<span class="material-symbols-outlined">api</span>' %}
+
+{{ site.alert.secondary }}
+{{pkg-icon}} Flutter package<br>
+&nbsp;&nbsp;Packages included in the templates in **bold**<br>
+{{api-icon}} API documentation<br>
+{{codelab}} Codelab <br>
+{{recipe-icon}} Cookbook recipe<br>
+{{tool-icon}} Desktop application<br>
+{{assets-icon}} Game assets<br>
+{{engine}} Game engine<br>
+{{doc-icon}} Guide<br>
```

Comment:
Indent each line by 2 spaces. Why not just use a bulleted list?

Reply:
Done. It looked weird to have a bullet, then an icon next to it.